uploader/wiki_uploader.py

```diff
@@ -32,7 +32,7 @@ def upload_file(self, file_name, file_stream, description=""):
             filename=file_name,
             description=description,
             ignore=True,
-            comment="Uploaded with Google drive to commons.",
+            comment=get_initial_page_text(datecreated, description),
```

Comment:
datecreated is not available here. I think you forgot to declare it ?

Reply:
and lets make it date_created, please ?

Reply:
yeah sure. I will do that

Reply:
I have made the necessary changes and could you please review it.